.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: PlatformIO CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example: [examples/Ex_01_SayHello/Ex_01_SayHello.ino]
```

Comment:
The build is failing because it is not picking up the library dependencies:
https://github.com/thijstriemstra/ArduinoProcessScheduler/runs/1780937482